### PR TITLE
JDK-8263235 : sanity/client/SwingSet/src/ColorChooserDemoTest.java failed throwing java.lang.NoClassDefFoundError

### PR DESCRIPTION
**Pull request by amresh-sahu**
Added @build org.jemmy2ext.JemmyExt.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8263235](https://bugs.openjdk.java.net/browse/JDK-8263235): sanity/client/SwingSet/src/ColorChooserDemoTest.java failed throwing java.lang.NoClassDefFoundError

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.java.net/census#psadhukhan) (@prsadhuk - **Reviewer**)
 * [Sergey Bylokhov](https://openjdk.java.net/census#serb) (@mrserb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/2958/head:pull/2958` \
`$ git checkout pull/2958`

Update a local copy of the PR: \
`$ git checkout pull/2958` \
`$ git pull https://git.openjdk.java.net/jdk pull/2958/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2958`

View PR using the GUI difftool: \
`$ git pr show -t 2958`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/2958.diff">https://git.openjdk.java.net/jdk/pull/2958.diff</a>

</details>

### CODE REVIEW

**Comment by amresh-sahu** [comment]
@shurymury , @akolarkunnu 
Kindly review my changes.

**Comment by akolarkunnu** [comment]
looks good to me

File: test/jdk/sanity/client/SwingSet/src/ColorChooserDemoTest.java [hunk 1]
```diff
@@ -62,6 +62,7 @@
 * @library /sanity/client/lib/SwingSet3/src
 * @modules java.desktop
 *          java.logging
+* @build org.jemmy2ext.JemmyExt
 * @build com.sun.swingset3.demos.colorchooser.ColorChooserDemo
```
Comment (prsadhuk) [inline test/jdk/sanity/client/SwingSet/src/ColorChooserDemoTest.java:66]: I guess you also need to update GridBagLayoutDemoTest.java with the same.

**Review by prsadhuk — APPROVED** [review]

**Review by mrserb — APPROVED** [review]

**Pull request closed without merging**